app/modules/kit-library/assets/js/models/taxonomy.js

```diff
@@ -1,6 +1,45 @@
 import BaseModel from './base-model';
+import { TIERS } from 'elementor-utils/tiers';
+import { __ } from '@wordpress/i18n';
 
-export const taxonomyType = [
+const FREE = 'free',
+	ESSENTIAL = 'essential',
+	ADVANCED = 'advanced',
+	PRO = 'pro',
+	EXPERT = 'expert,',
+	AGENCY = 'agency';
+
+export const SUBSCRIPTION_PLAN = 'subscription_plans';
+
+export const OLD_PLAN_TEXTS = {
+	[ FREE ]: __( 'Free', 'elementor' ),
+	[ PRO ]: __( 'Pro', 'elementor' ),
+	[ ADVANCED ]: __( 'Advanced', 'elementor' ),
+	[ EXPERT ]: __( 'Expert', 'elementor' ),
+	[ AGENCY ]: __( 'Agency', 'elementor' ),
+};
+
+export const NEW_PLAN_TEXTS = {
+	[ FREE ]: __( 'Free', 'elementor' ),
+	[ ESSENTIAL ]: __( 'Essential', 'elementor' ),
+	[ ADVANCED ]: __( 'Advanced & Higher', 'elementor' ),
+};
+
+const TAXONOMY_TRANSFORM_MAP = {
```

Comment:
is this a different TAXONOMIES?  
```

export const TAXONOMIES = [
	{
		key: 'categories',
```

Reply:
WDYM?

Reply:
We needed to take the original 5 plans, and divide it into 3 new ones